.circleci/config.yml

```diff
@@ -8,105 +11,98 @@ references:
           TARGET_ARCH: x64
     working_directory: ~/neon-wallet
 
-  win_config: &win_config
-    docker:
-      - image: electronuserland/builder:wine
-        environment:
-          TARGET_ARCH: x64
-    working_directory: ~/neon-wallet
-
   workspace_root: &workspace_root ~/neon-wallet
 
   attach_workspace: &attach_workspace
     attach_workspace:
       at: *workspace_root
 
 jobs:
-  build:
+  test:
     <<: *container_config
     steps:
       - checkout
       - restore_cache:
           key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
       - run: apt-get -y update
-      - run: apt-get -y install libusb-1.0-0-dev icnsutils graphicsmagick libudev-dev
       - run: yarn config delete proxy
       - run: yarn install --ignore-engines --frozen-lockfile --network-timeout 1000000 --network-concurrency 1
       - run: yarn lint
       - run: yarn flow
       - run: yarn assets
+      - run: yarn test-ci
       - save_cache:
           key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
           paths:
             - /usr/local/share/.cache/yarn/v2
             - ./node_modules
 
-  test:
+  build_linux:
     <<: *container_config
     steps:
       - checkout
       - restore_cache:
           key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
       - run: apt-get -y update
-      - run: apt-get -y install libusb-1.0-0-dev graphicsmagick libudev-dev
+      - run: apt-get -y install libusb-1.0-0-dev icnsutils graphicsmagick libudev-dev
       - run: apt-get -y install libxtst6 libxss1 libgtk2.0-0 libnss3 libasound2 libgconf-2-4
-      - run: yarn test-ci
-    store_artifacts:
-      - path: artifacts/
-      - destination: yarnpkg
+      - run: yarn config delete proxy
+      - run: yarn install --ignore-engines --frozen-lockfile --network-timeout 1000000 --network-concurrency 1
+      - run: yarn dist
+      - store_artifacts:
+          path: dist
+          destination: build
 
-  deploy_win64:
-    <<: *win_config
+  build_macos:
+    macos:
+      xcode: "12.0.0"
     steps:
       - checkout
-      - restore_cache:
```

Comment:
I couldnt figure out how to make caching work as there was some strange cryptic error occurring in circleCI when pulling down the yarn cache... We should probably revisit this at a later date to reduce our build time